service/chartconfig/v1/helm/helm.go

```diff
@@ -55,8 +51,8 @@ func (c *Client) GetReleaseContent(customObject v1alpha1.ChartConfig) (*Release,
 	releaseName := key.ReleaseName(customObject)
 
 	resp, err := c.helmClient.ReleaseContent(releaseName)
-	if strings.HasPrefix(err.Error(), notFoundErrorPrefix) {
-		return nil, microerror.Maskf(notFoundError, "Release: %s not found", releaseName)
+	if IsReleaseNotFound(err) {
+		return nil, microerror.Maskf(releaseNotFoundError, "%s", releaseName)
```

Comment:
We can just put the release name variable. No need for the format string here unless `releaseName` is not already a string.